todoman/model.py

```diff
@@ -237,7 +237,7 @@ class Database:
     def __init__(self, path):
         self.path = path
 
-    @cached_property
+    @property
```

Comment:
You'll need to manually cache this to avoid the extra overhead.

That is:
* Set `_cached_todos` on first read.
* Manually add new entries there.
* Manually remove entries from there.
* Return `_cached_todos` if it's defined, otherwise cached all of them.